README.md

```
@@ -92,7 +92,7 @@ greencity.bots.ubs-bot-token=${TELEGRAM_BOT_TOKEN}
 
 2. Run GreenCityUser project (look up paragraph [How to run](https://github.com/ita-social-projects/GreenCityUser#42-how-to-run)).
 
-3. Use the following link to open Swagger UI: http://localhost:8060/swagger-ui.html#/
+3. Use the following link to open Swagger UI: http://localhost:8050/swagger-ui.html#/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)